content/en/docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -60,6 +60,13 @@ duration as `--node-status-update-frequency`.
 
 The behavior of the static policy can be fine-tuned using the `--cpu-manager-policy-options` flag.
 The flag takes a comma-separated list of `key=value` policy options.
+This feature can be disabled completely using the `CPUManagerPolicyOptions` feature gate.
+
+The policy options are split in two groups: alpha quality, hidden by default, and beta quality,
```

Comment:
nit: s/in/into

Reply:
Also, maybe add parentheses around the visibility consstraints to make them more clear with the commas, i.e.:
> The policy options are split into two groups: alpha quality (hidden by default), and beta quality (visible by default).

Reply:
ack to both.